test/integration/lerna-publish.test.js

```diff
@@ -31,6 +31,21 @@ describe("lerna publish", () => {
     });
   });
 
+  test.concurrent("validates --cd-version", () => {
+    return initFixture("PublishCommand/normal").then((cwd) => {
+      const args = [
+        "publish",
+        "--skip-npm",
+        "--cd-version=nope",
```

Comment:
Try omitting `--cd-version` entirely, and you'll see why I had to manually validate it.

I've already noted this disparity upstream: https://github.com/yargs/yargs/issues/849